### ISSUE TITLE
Console shows orion.editor: Line 11 : Col 2

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1196]
@peller

When I use the source editor, I see console.log messages such as:

```
orion.editor: Line 11 : Col 2
```

Is this our code or the Orion code?

**Comment by peller** [comment]
Orion.  Well, sort of.

Two potential fixes: 1) get Orion to remove the log (I wouldn't recommend we diverge from their code) or 2) change our status reporter to ignore logs and report only errors -- see davinci/ui/Editor.js: _createEditor()

``` js
            options = {
                statusReporter: function(message, isError) {
                    var method = isError ? "error" : "log";
                    console[method]("orion.editor: " + message);
                },
```

I didn't want to turn off the logs right away...

**Comment by waynevicknair** [comment]
Fixed with commit be09dd6c5dd0db9eb488e54b02bd122eb5e90d0d -- made statusReporter only display errors in console.
